dali/python/nvidia/dali/types.py

```diff
@@ -16,7 +16,8 @@
 from enum import Enum, unique
 import re
 
-from nvidia.dali.backend_impl.types import DALIDataType, DALIImageType, DALIInterpType
+from nvidia.dali.backend_impl.types import DALIDataType, DALIImageType, DALIInterpType, \
+    DALIBorderType
```

Comment:
I'd vote for not exposing it at all. If you decide on exposing it anyway, it probably should land in  `_enum_types` list too.

Reply:
After consideration, I'm going to remove the `DALIBorderType`.